src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -496,7 +495,9 @@ def __call__(
         extra_step_kwargs = self.prepare_extra_step_kwargs(generator, eta)
 
         # 7. Denoising loop
-        for i, t in enumerate(self.progress_bar(timesteps)):
+        i = 0
+        t = self.scheduler.timesteps[0]
+        while t > 0:
```

Comment:
We need to keep `i` and `t`, which is very ugly. `i` is only required for the callback.